imap_processing/tests/swapi/test_swapi_l1.py

```diff
@@ -79,7 +79,7 @@ def test_decompress_count():
     # compressed + no-overflow, compressed + overflow, no compression
     raw_values = np.array([[12, 0xFFFF, 12]])
     compression_flag = np.array([[1, 1, 0]])
-    expected = np.array([[12 * 16, -1, 12]])
+    expected = np.array([[12 * 16, np.iinfo(np.int32).max, 12]], dtype=np.int32)
     returned_value = decompress_count(raw_values, compression_flag)
     np.testing.assert_array_equal(returned_value, expected)
 
```

Comment:
It would be helpful to have one test that shows the flags set in dataset["swp_flags"].

Reply:
for sure! I will add a test for this.

Reply:
Laura, actually I am going to wait to add the test and do this in future PR if you don't mind. Right now, all flags are set to 0 but SWAPI will provide me some useful data that will have some good flags set. I will add the test with that change.